clash-ghc/src-ghc/Clash/GHC/GenerateBindings.hs

```diff
@@ -142,6 +143,9 @@ generateBindings useColor primDirs importDirs dbs hdl modName dflagsM = do
   let prepStartDiff = reportTimeDiff prepTime startTime
   putStrLn $ "Clash: Parsing and compiling primitives took " ++ prepStartDiff
 
+  -- TODO: Just a temporary measure to see the RecInfo
+  print (mkRecInfo allBindings)
+
```

Comment:
^

Reply:
I think this should have a TODO that we eventually want to get this information from:
https://github.com/clash-lang/clash-compiler/blob/305427f3e2bb44669ae971c4d0e34da6a7818ed0/clash-ghc/src-ghc/Clash/GHC/GenerateBindings.hs#L154-L197
And that we stop flattening recursive groups.